evalml/pipelines/time_series_regression_pipeline.py

```diff
@@ -161,3 +169,62 @@ def get_forecast_predictions(self, X, y):
         pred_dates = pd.DataFrame(self.get_forecast_period(X))
         preds = self.predict(pred_dates, objective=None, X_train=X, y_train=y)
         return preds
+
+    def get_prediction_intervals(
+        self,
+        X,
+        y=None,
+        X_train=None,
+        y_train=None,
+        coverage=None,
+    ):
+        """Find the prediction intervals using the fitted regressor.
+
+        This function takes the predictions of the fitted estimator and calculates the rolling standard deviation across
```

Comment:
this isn't true for all models right? Only the linear models? We should probably update this to point users to the estimator description.